adoc/retail_chap_formulas.adoc

```diff
@@ -111,69 +111,35 @@ Use only alphanumeric characters for the branch identifier.
 [[retail.sect.formulas.saltboot]]
 == Saltboot Formula
 
-The Saltboot formula is used to configure the boot image on a terminal.
+The Saltboot formula is used to configure disk images and partitioning.
 
-[NOTE]
-====
-There are many different configurations possible.
-This procedure outlines a simple, single-disk, three-partition configuration.
-Adjust it to suit your own environment.
-====
 
 
 .Procedure: Configuring the branch server with saltboot
 
-. In the {susemgr} {webui}, open the details page for the branch server, and navigate to the [guimenu]``Formulas`` tab￼.
```

Comment:
Please do not remove this, this is valid. Just move it down where Bind formula is talked about.

Reply:
OK, I think I understand. Let me know?